contracts/utils/EnumerableSet.sol

```diff
@@ -20,58 +20,60 @@ pragma solidity ^0.6.0;
 library EnumerableSet {
 
     struct AddressSet {
-        // Position of the value in the `values` array, plus 1 because index 0
-        // means a value is not in the set.
-        mapping (address => uint256) index;
-        address[] values;
+        address[] keys;
```

Comment:
I do not like this renaming to `keys`... They aren't keys in a set, they are values. I also don't understand what this has to do with maps.

Reply:
That was also my original intuition, but I think the consistency with `EnumerableMap` (where key and value have very clear meanings) is worth it.

You can think of the set as a map where the keys have no associated value.

Reply:
It's actually not so much a matter of consistency but rather avoiding confusion between the two.

Reply:
I don't see any possible confusion. When would people be confused or make a mistake because of confusion?

Reply:
A set would have `contains(value)` while a map `contains(key)`, despite them both being pretty much the same thing under the hood.

Something similar happens with `remove(value)` for set and `remove(key)` for map.

Reply:
I've been convinced and renamed to `values`